docs/pages/bare/installing-updates.mdx

```diff
@@ -29,7 +29,7 @@ Once installation is complete, apply the changes from the following diffs to con
 Modify the `expo` section of **app.json**. (If you originally created your app using `npx react-native init`, you will need to add this section.)
 The changes below add the updates URL to the Expo configuration.
 
-> The example URL shown here is for use with a custom update server running on the same machine. When using EAS Update, the EAS CLI will set this URL correctly for the EAS Update service.
+> The example updates URL and project ID shown here is for use with EAS Update, where the EAS CLI will set this URL correctly for the EAS Update service when running `eas update:configure`.
```

Comment:
```suggestion
> The example `updates` URL and `projectId` shown below are for use with EAS Update. The EAS CLI sets this URL correctly for the EAS Update service when running `eas update:configure`.
```

- Let's split this sentence for improved readability.
- Also, highlight `updates` and `projectId` so they can be directly referenced in the example below
- Fix grammatical error